app/src/editor/shared/components/SearchPanel/SearchPanel.pcss

```diff
@@ -27,9 +27,8 @@
 
 .SearchPanel .Container {
   height: 100%;
-  display: grid;
-  grid-template-columns: 1fr;
-  grid-template-rows: auto auto 1fr;
+  display: flex;
+  flex-direction: column;
```

Comment:
This fixes an issue where content is not clickable after clearing search results. Seems to be a bug as the problem doesn't exist in Chrome Canary 77.